builtin/providers/docker/resource_docker_container.go

```diff
@@ -101,6 +101,14 @@ func resourceDockerContainer() *schema.Resource {
 				Set:      resourceDockerPortsHash,
 			},
 
+			"extra_hosts": &schema.Schema{
```

Comment:
If this is a set, it's valid to have multiple instances of it. Might this be better named `extra_host` (singular) or even `host` such that the set semantics read better?